pretty_gpx/common/drawing/elevation_stats_section.py

```diff
@@ -11,6 +11,7 @@
 from pretty_gpx.common.gpx.gpx_track import GpxTrack
```

Comment:
Hi @JulesL2  :) 

I fixed the rendering of large city runs in PR https://github.com/ThomasParistech/pretty-gpx/pull/51 :tada: 
You'll have some conflicts

Reply:
@ThomasParistech  Everything is up to date now 🥳 !